### ISSUE TITLE
Please Update the Library to Support Angular 14

### ISSUE THREAD

**Issue by AshotAleqs** [denniske/ngx-translate-multi-http-loader#12]
Hello,  Please update The Package dependencies and dev dependencies to Support Angular 14.

Thanks

**Comment by denniske** [comment]
Hi please try v8.0.0

**Comment by AshotAleqs** [comment]
Hello @denniske can you please check it when you get a chance, v8.0.1 is throwing an error

**Comment by denniske** [comment]
What is the error?

**Comment by tehXor** [comment]
> What is the error?

```
error TS2307: Cannot find module 'ngx-translate-multi-http-loader' or its corresponding type declarations.
import {MultiTranslateHttpLoader} from 'ngx-translate-multi-http-loader'
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This is thrown in the app module which stayed unchanged and is basically exactly like in the https://github.com/denniske/ngx-translate-multi-http-loader/blob/master/README.md. Obviously `ngx-translate-multi-http-loader` is actually installed and exists in `node_modules`.

Angular 14 + `v7.0.5` seems to work regardless of the peer dependency warning, though. (But I haven't tested much.)

**Comment by AshotAleqs** [comment]
Hello @denniske, sorry crazy week. I didn't have time to reinstall and send the error to you. Please have a look at the above error

**Comment by denniske** [comment]
Hi, please check this repo: https://github.com/denniske/ngx-translate-multi-http-loader-demo-14

It runs `npm start` and `npm run build` without errors.

**Comment by AshotAleqs** [comment]
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/48962964/174083906-79f2537c-70c4-4202-ab0d-4622e547c2ad.png">

IHi @denniske, here is the error message that I am getting, even after reinstalling `node_modules`

**Comment by denniske** [comment]
can you start the application without errors? in that case a `// @ts-ignore` in the line above the import might help.

**Comment by AshotAleqs** [comment]
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/48962964/174086003-eacd2a78-2b45-4c2e-9db7-a37669e9713e.png">

after `// @ts-ignore` I am getting the above error

**Comment by denniske** [comment]
I am not sure what is the issue but I think it might be the easiest if you just copy the code from https://github.com/denniske/ngx-translate-multi-http-loader/blob/master/src/multi-http-loader.ts into your project instead using the library using npm to solve this issue.

**Comment by AshotAleqs** [comment]
@denniske  usually I am getting this kind of error when mistakenly publishing the library from `/root` not from `/dist`, is there any way to double-check if you published the library from `/dist`?

**Comment by AshotAleqs** [comment]
Thanks, @denniske, I did it already, just I saw your package is popular, and wanted to notify you about the issue.

**Comment by denniske** [comment]
I see, thanks for opening the issue :)

I have ran `npx tsc && npm publish --access=public` and in the node_modules folder of my demo project I can see the generated *.d.ts file in the dist folder. So I think it is correctly published.

<img width="486" alt="image" src="https://user-images.githubusercontent.com/1076211/174089540-748d9454-178b-442d-be41-3310fc5d58fe.png">

**Comment by AshotAleqs** [comment]
Hmm, strange... we are using your package in several projects, and here is the library structor from 7.x version 
<img width="560" alt="image" src="https://user-images.githubusercontent.com/48962964/174091105-c61cf68d-b5c7-408c-81b2-111a0b18024b.png">

**Comment by denniske** [comment]
Yeah let me explain. So basically I just want to package this one file I mentioned earlier into an npm package. But with angular this has taken me so much effort to get it right: On every angular update I had to update so many dependencies / change so much stuff that I often just recreated the whole library.

So in the end I thought there has to be a simpler way and just put the one file into an npm package instead building an angular lib. And as it worked for me in the demo project I thought everything is fine.

But if this approach does not work, I am happy to give the project to another maintainer. I don't have the bandwidth to make it an angular lib anymore.

**Comment by tehXor** [comment]
@denniske thanks for your explanation, great simple concept for a multi loader and maintenance thus far! 👍👍

I pulled the multi loader class directly into my project were I'll continue to use and maintain it. I can so much understand that all this required maintenance to keep this project a public npm package is not worth the effort. (Therefore I also wouldn't like to take over.)

It's a pity, though. There should be a shift which reduces dumb boilerplate work for package maintainers which each and every update and increases responsibility for project maintainers who use it. But probably this is against the goal that everybody can "easily" manage a project after a 30 minute YouTube tutorial (however well this may turn out) 😅

**Comment by AshotAleqs** [comment]
Agreed with @tehXor, Thanks for your time and support. Yeah, I have published only one library and understood, that the support will be a problem, And I am not sure I will have much time to take over this package as well.

**Comment by it11ah1** [comment]
from package.json
```
  "main": "dist/index.js",
  "types": "dist/index.d.ts",
```

**Comment by denniske** [comment]
Merged PR #13 from @almarro1 and published v8.0.2. Please check if it works now.

**Comment by AshotAleqs** [comment]
Thank you very much @denniske, I can confirm the problem is fixed on `v8.0.2`
